sources/packages/api/tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "test", "dist", "**/*spec.ts", "scripts", "__test__", "testHelpers", "ormconfig.js", "env_setup.js", "**/__mocks__/*"]
+  "exclude": ["node_modules", "test", "dist", "**/*spec.ts", "scripts", "__test__", "testHelpers", "ormconfig.ts", "env_setup.js", "**/__mocks__/*"]
```

Comment:
Does `ormconfig.ts` need to be ignored? It is a typescript file to be compiled/transpiled as the others, right?

Reply:
Valid point. Speaking beyond the context of the PR, IMO **/*spec.ts also shouldn't be excluded.

Reply:
The spec files are not part of the PROD package, why should we keep them?
Sorry, I am not following.